connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -84,6 +84,11 @@ public abstract class MirrorConnectorConfig extends AbstractConfig {
     public static final String REPLICATION_POLICY_SEPARATOR_DEFAULT =
             MirrorClientConfig.REPLICATION_POLICY_SEPARATOR_DEFAULT;
 
+    private static final String INTERNAL_TOPIC_SEPARATOR_ENABLED =  MirrorClientConfig.INTERNAL_TOPIC_SEPARATOR_ENABLED;
+    private static final String INTERNAL_TOPIC_SEPARATOR_ENABLED_DOC = "whether or not to use replication.policy.separator to control internal topic names";
```

Comment:
IMO it's better to make `MirrorClientConfig.INTERNAL_TOPIC_SEPARATOR_ENABLED_DOC` public and then use that same value here:
```suggestion
private static final String INTERNAL_TOPIC_SEPARATOR_ENABLED_DOC = MirrorClientConfig.INTERNAL_TOPIC_SEPARATOR_ENABLED_DOC;
```

I know we don't do that for the rest of the fields in this class that we borrow from `MirrorClientConfig` but it'll be easier to keep the docstrings in sync this way and apart from a small amount of inconsistency, there don't seem to be any drawbacks.